app/js/lib/server/webpush.js

```diff
@@ -31,32 +31,24 @@ export default class WebPush extends EventDispatcher {
       this[p.listenForMessages].bind(this));
 
     return navigator.serviceWorker.ready
-      .then((reg) => {
-        reg.pushManager.getSubscription()
-          .then((existing) => {
-            if (existing && !resubscribe) {
-              // nothing left to do
-              // @todo We should check if we have this subscription on the
-              // server, if not re-register it on calendar's server.
-              // https://github.com/fxbox/calendar/issues/20
-              return;
-            }
+      .then(
+        (reg) => reg.pushManager.getSubscription()
```

Comment:
I think this is the code in https://cs.chromium.org/chromium/src/content/browser/push_messaging/push_messaging_message_filter.cc?q=PushMessagingHostMsg_Subscribe&sq=package:chromium&dr=C&l=283

That said maybe we should not rely on this, it's just as easy to do what I do here.